stdlib/jdbc/src/main/ballerina/src/java.jdbc/client.bal

```diff
@@ -33,6 +37,11 @@ public type Client client object {
     # + parameters - The parameters to be passed to the procedure/function call
     # + return - A `table[]` if there are tables returned by the call remote function and else nil,
     #            `Error` will be returned if there is any error
+    #
+    # # Deprecated
+    # New experimental jdbc2 module is introduced in 1.2 release and it will be fully supported with in 1.3 release
```

Comment:
```suggestion
    # A new experimental JDBC2 module is introduced in the Ballerina 1.2 release and it will be fully supported by the 1.3 release.
```